grafana/dashboards/cmode/volume.json

```diff
@@ -89,14 +89,31 @@
     }
   ],
   "panels": [
+    {
+      "datasource": "${DS_PROMETHEUS}",
+      "gridPos": {
+        "h": 3,
+        "w": 24,
+        "x": 0,
+        "y": 0
+      },
+      "id": 91,
+      "options": {
+        "content": "Workload volume metrics are more useful for NAS and detailed view would be available in QoS row. <br>Refer ONTAP Performance - https://kb.netapp.com/on-prem/ontap/Perf/Perf-KBs/What_is_the_difference_in_latency_of_the_same_volume_between_statistics_volume_show_and_qos_statistics_volume_latency_show_commands",
```

Comment:
Volume latencies are missing some latencies from NAS protocols. Instead, use the workload volume metrics in the QoS row for a more detailed breakdown.

[Details in ONTAP Performance KB article](https://kb.netapp.com/on-prem/ontap/Perf/Perf-KBs/What_is_the_difference_in_latency_of_the_same_volume_between_statistics_volume_show_and_qos_statistics_volume_latency_show_commands)